.env.example

```diff
@@ -5,4 +5,8 @@ INFURA_KEY=infura-key
 INFURA_JWT_ID=infura-jwt-id
 DATABASE_USERNAME=bff-db-user
 DATABASE_PASSWORD=bff-db-password
-DATABASE_NAME=bff-db
\ No newline at end of file
+DATABASE_NAME=bff-db
+ORDERBOOK_DATABASE_HOST=
```

Comment:
ubernit: pick we can always add some comment space things out, group them, and give more context